Makefile

```diff
@@ -92,6 +107,13 @@ restore-postgres: terraform-init read-deployment-config ## make dev restore-post
 	echo "Restoring ${POSTGRES_DATABASE_NAME} from $(TF_VAR_paas_restore_db_from_db_instance) before $(TF_VAR_paas_restore_db_from_point_in_time_before)"
 	make ${DEPLOY_ENV} terraform-apply
 
+restore-data-from-backup: read-deployment-config # make dev restore-data-from-backup CONFIRM_RESTORE=YES
```

Comment:
Comment should include `BACKUP_FILENAME="find-a-lost-trn-production-pg-svc-yyyy-mm-dd-HH` as this is a required arg